@shared/api/internal.ts

```diff
@@ -2,7 +2,8 @@ import StellarSdk from "stellar-sdk";
 import { DataProvider } from "@stellar/wallet-sdk";
 import {
   Account,
-  AccountDetailsInterface,
+  AccountBalancesInterface,
+  AccountHistoryInterface,
```

Comment:
I went ahead and split AccountDetails into Balances (for Account) and History (for Account History)